app/routes/events/view/index.js

```diff
@@ -14,18 +14,30 @@ export default class extends Route.extend(EmberTableRouteMixin) {
       'page[number]' : params.per_page || 1
     };
     queryString = this.applySortFilters(queryString, params);
+    const sponsorsPromise = this.asArray(eventDetails.query('sponsors', {}, queryString));
+    const roleInvitesPromise = eventDetails.query('roleInvites', {});
+    const sessionTypesPromise = eventDetails.query('sessionTypes', {});
+    const socialLinksPromise = eventDetails.query('socialLinks', {});
+    const statisticsPromise = eventDetails.query('generalStatistics', {});
+    const orderStatPromise = eventDetails.query('orderStatistics', {});
+    const ticketsPromise = eventDetails.query('tickets', {});
+    const rolesPromise = this.store.findAll('role');
+
+    const [sponsors, roleInvites, sessionTypes, socialLinks,
+      statistics, orderStat, tickets, roles] = await Promise.all([sponsorsPromise, roleInvitesPromise, sessionTypesPromise, socialLinksPromise,
```

Comment:
`allSettled` is better because `all` throws error when any of the promises reject, whereas `allSettled` waits for all to resolve and gives the status

Also, please use RSVP's allSettled implementation